tests/test-mover-included.c

```diff
@@ -1,19 +1,23 @@
-#include<stdio.h>
+/*
+get-ir:
+        status: success
+split:
+        status: success
+build:
+        status: success
+run:
+        stdout: 14 10
+        status: 24
```

Comment:
Conventionally we put the `status` before `stdout` (not a big deal but we might as well do it).

Reply:
Fixed in: e422905